### DISCUSSION TITLE
MOD3.0: Design issue #4: What class (outside of DCAT) can we consider a Semantic Artefact generalize?

### DISCUSSION THREAD

**Discussion by jonquet** [sifrproject/MOD-Ontology#37] [General]
We reviewed previous metadata vocabularies that describe some kind of semantic artefact: OWL, SKOS, OMV, VOAF, MOD, DOOR

- We propose to re-integrate in this model previous vocabularies: OMV, DOOR, VOAF and previous MOD
Then redefine the property in the mod namespace and abandoned all the ones that were duplicates from other metadata vocabularies e.g., omv:license
- Shall we extend the SA class?

<img width="488" alt="image" src="https://user-images.githubusercontent.com/7379183/231009326-a73a8e04-852f-4ce5-8cd7-eac0b490db16.png">

**Comment by jonquet** [comment]
Properties from the incorporated vocabularies

From VOAF:
<img width="475" alt="image" src="https://user-images.githubusercontent.com/7379183/231009423-cffe8b6c-09a4-44b2-a5af-7b96a43d91a1.png">

From DOOR: 
<img width="591" alt="image" src="https://user-images.githubusercontent.com/7379183/231009472-84bd517f-5703-4453-99c5-4e3ce5d1d49a.png">

From MOD (previous version):
<img width="298" alt="image" src="https://user-images.githubusercontent.com/7379183/231009505-b855be0f-99b9-48b0-a388-022d452ae827.png">

From OMV:
<img width="424" alt="image" src="https://user-images.githubusercontent.com/7379183/231009541-a85a85ac-9fc1-4dfd-9225-5bb6a6f1e393.png">

**Comment by nichtich** [comment]
The [NKOS KOS Types Vocabulary](http://bartoc.org/en/node/1665) has been defined in 2014 based on research in knowledge organization. "Semantic Artifact" is an alias for [Knowledge Organization System](http://www.wikidata.org/entity/Q6423319) (KOS). See [this encyclopedia article](https://www.isko.org/cyclo/kos) for a lengthy description, including a typology of KOS aka semantic artifacts.

The [NKOS KOS Types Vocabulary](https://bartoc.org/en/node/1665) is used to classify more than 3500 semantic artifacts in BARTOC. I wonder why you don't use this established list. Or just follow the [SKOS Primer](https://www.w3.org/TR/skos-primer/) an make everything subclasses of `skos:ConceptScheme`. I'd better remove `mod:Taxonomy`, `mod:Thesaurus`, and `mod:Terminology` from MOD 3.0 and stick to definition of properties of semantic artifacts. If you ask two experts about difference between any of the KOS types you get three answers at least.

> We reviewed previous metadata vocabularies that describe some kind of semantic artefact: OWL, SKOS, OMV, VOAF, MOD, DOOR

What about [NKOS AP](https://nkos.dublincore.org/nkos-ap.html), [JSKOS](https://gbv.github.io/jskos/), and [XKOS](https://rdf-vocabulary.ddialliance.org/xkos.html)?

**Comment by jonquet** [comment]
Dear Jakob @nichtich , thanks for feedback and comments.
Some of the elements in this thread (despite being displayed from April 2023) have been open questions for long time and not real decision was taken out of this. Mostly because proposing a new standard with "super classes" over classes in standards already well established (OWL and SKOS here) is a bit tricky. Origninaly the idea was just to draw where was the object SemanticArtefact for us with respect to owl:Ontology and skos:ConceptScheme ... and from a drawing one day some lines were formalize in mod.ttl 
(https://github.com/FAIR-IMPACT/MOD/blob/7e590b63a80b3fa01f62d97897241fb5b8943852/mod.ttl#L471)

So with respect to the points :
- Making everything subclasses of skos:ConceptScheme why not, can you point me to a precise point in the primer that would suggest this. 
- Remove mod:Taxonomy, mod:Thesaurus, and mod:Terminology from MOD 3.0 ...as they are not really bringing anything, I do agree also.
- Plug-in intelligently NKOS KOS Types Vocabulary within MOD, yes This would be very good.

MOD is open and we spend less time on this that what we should. I would recommend open another issue for the first item to discuss this more and make a couple of pull requests to mod.ttl for the 2 other items. 

>What about [NKOS AP](https://nkos.dublincore.org/nkos-ap.html), [JSKOS](https://gbv.github.io/jskos/), and [XKOS](https://rdf-vocabulary.ddialliance.org/xkos.html)?

Could you open an issue for each of this vocabulary that would list the properties we should "incorportate" in MOD and on which class of MOD. I am sure we can improve this.

**Comment by nichtich** [comment]
> Making everything subclasses of skos:ConceptScheme why not, can you point me to a precise point in the primer that would suggest this.

[SKOS Primer](https://www.w3.org/TR/skos-primer/) starts with

> SKOS—Simple Knowledge Organization System—provides a model for expressing the basic structure and content of concept schemes such as thesauri, classification schemes, subject heading lists, taxonomies, folksonomies, and other similar types of controlled vocabulary. 

This does subsume Taxonomy, Thesaurus, and Terminology but ontologies are not listed explicitely.

**Reply by jonquet** [reply]
Yes indeed, this is essentially the reason.
We can't have the statement in MOD: 
`owl:Ontology rdfs:subClassOf skos:ConceptScheme` because we do not own neither OWL nor SKOS. 

But we could possibly (if folks arround MOD agree with this) add the statements: 
`owl:Ontology rdfs:subClassOf mod:SemanticArtefact`
`skos:ConceptScheme rdfs:subClassOf mod:SemanticArtefact`
If this si ok I am fine with adding these 2 "important" statements has they are "putting us above"

**Reply by nichtich** [reply]
I think you meant `mod:SemanticArtifact` instead of `mod:ConceptScheme`? I'd also welcome adding

~~~ttl
owl:Ontology rdfs:subClassOf mod:SemanticArtefact
skos:ConceptScheme rdfs:subClassOf mod:SemanticArtefact
~~~

By the way this also states both classes are subclass of [`dcat:Resource`](https://www.w3.org/TR/vocab-dcat-3/#Class:Resource) for good reason.

Thinking about examples of semantic artifacts *not* being instance of OWL or SKOS: a Knowledge Graphs exposed with Wikibase ontology in RDF typically neither uses OWL nor SKOS. Funny, I forgot it [was me to propose to include its metadata](https://phabricator.wikimedia.org/T178978). Wikidata is described by URI <https://www.wikidata.org/about#catalog> as instance of `dcat:Dataset', see [this query](https://w.wiki/BNdL).

**Reply by jonquet** [reply]
Sorry. Was in the middle of a meeting. I'll correct
